cluster_experiments/washover.py

```diff
@@ -10,6 +10,27 @@
 class Washover(ABC):
     """Abstract class to model washovers in the switchback splitter."""
 
+    def _validate_columns(
+        self,
+        df: pd.DataFrame,
+        truncated_time_col: str,
+        cluster_cols: List[str],
+        original_time_col: str,
+    ):
+        if original_time_col not in df.columns:
+            raise ValueError(
+                "The original_time_col is not defined and/or not available in the dataframe columns."
+            )
+        if truncated_time_col not in cluster_cols:
+            raise ValueError(
+                f"The truncated_time_col '{truncated_time_col}' is not in the cluster columns."
```

Comment:
```suggestion
                f"{truncated_time_col = } is not in the cluster columns."
```
wdyt of something like this? and similar for original_time_col and cluster_cols